src/lib.rs

```diff
@@ -206,8 +207,35 @@ pub fn can_message(address: &Address) -> bool {
 /// Get a capability in our store
 /// NOTE unfortunatly this is O(n), not sure if wit let's us do any better
 pub fn get_capability(our: &Address, params: &str) -> Option<Capability> {
+    let params = serde_json::from_str::<Value>(params).unwrap_or_default();
     crate::our_capabilities()
         .iter()
-        .find(|cap| cap.issuer == *our && cap.params == params)
+        .find(|cap| {
+            let cap_params = serde_json::from_str::<Value>(&cap.params).unwrap_or_default();
```

Comment:
Once we parse both strs into `serde_json::Value`, can't we do a direct equality check?

The old code compared the json-strings. Of course, that caused issues with JSON that parsed to the same `Value` but weren't exactly the same string. But once we parse to `Value`, simply checking for equality should work.